mkt/urls.py

```diff
@@ -41,6 +42,12 @@
         mini_langpack_manifest, name='langpack.manifest'),
     url('^extension/(?P<uuid>[0-9a-f]{32})/manifest.json$',
         mini_extension_manifest, name='extension.mini_manifest'),
+    url('^late_customization/%s/manifest.webapp$' % mkt.ADDON_UUID,
```

Comment:
So, we have a different mini-manifest. Is platform doing some transformation ? Or are we going to add a hack to fireplace to have it consider the app is installed no matter what manifest is used ?